lib/Frontend/ModuleInterfaceLoader.cpp

```diff
@@ -2193,7 +2193,9 @@ InterfaceSubContextDelegateImpl::runInSubCompilerInstance(StringRef moduleName,
 
   // FIXME: Hack for Darwin.swiftmodule, which cannot be rebuilt with C++
   // interop enabled by the Swift CI because it uses an old host SDK.
-  if (moduleName == "Darwin") {
+  // FIXME: Hack for CoreGraphics.swiftmodule, which cannot be rebuilt because
+  // of a CF_OPTIONS bug (rdar://142762174).
+  if (moduleName == "Darwin" || moduleName == "CoreGraphics") {
```

Comment:
Can we hoist this out of the compiler and into arguments? I suspect that there will be a number of other modules that need this behaviour (I can see submodules in WinSDK wanting this as well).

Reply:
Hmm, what would be the case where we would want to apply this to a WinSDK module?

This isn't intended to be something accessible via a compiler flag, since it might result in linker errors if you try using any of the problematic types from Swift. In this case the `CF_OPTIONS` types are triggering a compiler bug when C++ interop is enabled, which is a bug that we'll need to fix.